desktop/cmp/dash/container/impl/DashContainerUtils.ts

```diff
@@ -37,7 +38,7 @@ function convertGLToStateInner(configItems = [], contentItems = [], dashContaine
                 viewSpec = dashContainerModel.getViewSpec(viewSpecId),
                 viewModelId = getViewModelId(contentItem),
                 viewModel = dashContainerModel.getViewModel(viewModelId),
-                view = {type: 'view', id: viewSpecId};
+                view = {type: 'view', id: viewSpecId} as any;
```

Comment:
Does anyone else find this to be a rather odd requirement of the language, the way it "snaps" the shape of the initial assignment by default?